chaoscenter/web/src/views/ChaosHubMenu/__tests__/ChoasHubMenu.test.tsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import { render, fireEvent, screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import { MemoryRouter } from 'react-router-dom';
+import type { MutationFunction, ApolloQueryResult } from '@apollo/client';
+import type {
+  DeleteChaosHubResponse,
+  DeleteChaosHubRequest,
+  SyncChaosHubResponse,
+  SyncChaosHubRequest,
+  ListChaosHubRequest,
+  ListChaosHubResponse
+} from '@api/core';
+import { AuthType, type ChaosHub } from '@api/entities';
+import { ChaosHubMenuView } from '../ChaosHubMenu';
+
+// Mock external dependencies
+jest.mock('@harnessio/uicore', () => ({}));
+jest.mock('@blueprintjs/core', () => ({}));
+jest.mock('react-router-dom', () => ({
+  ...jest.requireActual('react-router-dom'),
+  useHistory: () => ({
+    push: jest.fn()
+  })
+}));
+jest.mock('@hooks', () => ({
+  useRouteWithBaseUrl: () => jest.fn()
+}));
+jest.mock('@strings', () => ({
+  useStrings: () => ({
+    getString: jest.fn().mockImplementation(key => key)
+  })
+}));
+jest.mock('@api/entities', () => ({}));
```

Comment:
These mocks could be avoided by using the `<DefaultLayoutTemplate>`, please refactor this test to update the same!